docs/source/frameworks/xgboost.rst

```diff
@@ -2,49 +2,174 @@
 XGBoost
 =======
 
-Users can now use XGBoost with BentoML with the following API: :code:`save_model`, :code:`load_model`, and :code:`get` as follows:
+This guide provides an overview of using `XGBoost <https://xgboost.readthedocs.io/en/stable/>`_ with BentoML.
+
+Compatibility
+~~~~~~~~~~~~~
+
+BentoML has been validated to work with XGBoost version 0.7post3 and higher.
+
+Saving a Trained Booster
+------------------------
+
+First, train or load a booster. In this example, we will be training a new booster using UCI's
+`breast cancer dataset <https://archive.ics.uci.edu/ml/datasets/breast+cancer+wisconsin+(diagnostic)>`_.
+If you've already saved a model using XGBoost, simply load it back into Python using
+``Booster.load_model``.
 
 .. code-block:: python
 
-   import bentoml
    import xgboost as xgb
+   from sklearn.datasets import load_breast_cancer
+
+   cancer = load_breast_cancer()
 
-   def xgboost_model() -> "xgb.Booster":
-      from sklearn.datasets import load_breast_cancer
+   X = cancer.data
+   y = cancer.target
 
-      # read in data
-      cancer = load_breast_cancer()
+   dt = xgb.DMatrix(X, label=y)
 
-      X = cancer.data
-      y = cancer.target
+   param = {"max_depth": 3, "eta": 0.3, "objective": "multi:softprob", "num_class": 2}
+   bst = xgb.train(param, dt)
 
-      dt = xgb.DMatrix(X, label=y)
 
-      # specify parameters via map
-      param = {"max_depth": 3, "eta": 0.3, "objective": "multi:softprob", "num_class": 2}
-      bst = xgb.train(param, dt)
+After training, use save_model to save the Booster instance to BentoML model store. XGBoost has no
+framework-specific save options.
```

Comment:
Could we bold/italic "no framework-specific" for better visibility?

Reply:
I think this is ok because I don't think a user would necessarily think there would be.

Maybe we should even remove that sentence? I think I was mostly trying to pad out the section because it felt so bare...

Reply:
lets remove the sentence, I don't think this provides any value here.